src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -73,6 +93,10 @@ const IncidentDetail = () => {
   const { get: getDefaultTexts, data: defaultTexts } = useFetch();
   const { get: getChildren, data: children } = useFetch();
 
+  const subcategories = useSelector(makeSelectSubCategories);
+
+  const handlingTimesBySlug = useMemo(() => getHandlingTimes(subcategories), [subcategories]);
```

Comment:
We could address this in another issue, the business really wan't everything to be loaded when the page is rendered, this will require some refactoring in how we handle subcategories on application.